auth/src/db/entities/users.entity.ts

```diff
@@ -0,0 +1,42 @@
+import { Expose } from 'class-transformer';
+import { Column, Entity, Index, PrimaryGeneratedColumn } from 'typeorm';
+
+@Index('email_unique', ['email'], { unique: true })
+@Index('users_pkey', ['pkUserId'], { unique: true })
+@Index('username_unique', ['username'], { unique: true })
+@Entity('users', { schema: 'public' })
+export class UserEntity {
+  @Expose()
+  @PrimaryGeneratedColumn({ type: 'integer', name: 'pk_user_id' })
+  pkUserId: number;
+
+  @Expose()
+  @Column('character varying', { name: 'username', unique: true, length: 24 })
+  username: string;
+
+  @Expose()
+  @Column('character varying', { name: 'email', unique: true, length: 128 })
+  email: string;
+
+  @Column('text', { name: 'password', nullable: true })
+  password: string | null;
+
+  @Expose()
+  @Column('character varying', {
+    name: 'status',
+    length: 16,
+    default: () => "'user'",
```

Comment:
like comment below, is this supposed to be?

Reply:
Like comment above, this was autogenerated using a tool. It certainly does not have destructive effects, so I will leave it in for now